lib/core/src/Network/Wai/Middleware/Logging.hs

```diff
@@ -257,8 +258,11 @@ instance ToText HandlerLog where
             path = T.decodeUtf8 $ rawPathInfo req
             query = T.decodeUtf8 $ rawQueryString req
         LogRequestBody ks body -> sanitize ks body
-        LogResponse time status -> mconcat [ code, " ", text, " in ", tsec ]
+        LogResponse time req status ->
+            mconcat [ method, " ", path, " ", code, " ", text, " in ", tsec ]
```

Comment:
Example in integration tests:
<img width="1259" alt="Skärmavbild 2021-03-22 kl  14 12 48" src="https://user-images.githubusercontent.com/304423/112204552-16854780-8c14-11eb-8d09-319b5939aa54.png">